docs/source/schema-delegation.md

```diff
@@ -193,8 +193,105 @@ Also provides the `info.mergeInfo.delegateToSchema` function discussed above.
 
 [Transforms](./schema-transforms.html) to apply to the query and results. Should be the same transforms that were used to transform the schema, if any. After transformation, `transformedSchema.transforms` contains the transforms that were applied.
 
+<h3 id="createRequest">createRequest</h3>
+
+The `createRequest` is a utility function for creating queries with multiple, aliased, roots and possible argument name collisions. The function should be called with these parameters:
+
+```js
+createRequest(
```

Comment:
Let's use an object of named parameters rather than positional parameters, please.